### PR TITLE
docs: revised Kotlin/JS

### PR DESCRIPTION
**Pull request by p7nov**
Revised the texts in the Kotlin/JS chapter.

### CODE REVIEW

File: examples/09_Kotlin_JS/06_HtmlBuilder.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # Html Builder
 
-This is an example of a Type-Safe Groovy-style Builder
-
-Builders are good for declaratively describing data in your code.
-In this example we show how to describe an HTML page in Kotlin
+Kotlin provides you with a option to describe structured data in a declarative style with _builders_.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/06_HtmlBuilder.md]: with 'an' option

Kotlin provides the option to describe structured data in a declarative style with _builders_.

File: examples/09_Kotlin_JS/02_js_function.md [hunk 2]
```diff
@@ -19,6 +32,6 @@ fun main(){
 
 </div>
 
-1. Create a JavaScript object literal. The `js(...)` function return type is `dynamic`.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/02_js_function.md]: I would leave it as Create as it is an instruction, or use the passive if it is a description so Creates

File: examples/09_Kotlin_JS/02_js_function.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ Should be used with extreme care.
...
+```kotlin
+fun main() {
+// sampleStart
+    js("alert(\"alert from Kotlin!\")") // 1
+// sampleEnd    
+}
+```
+</div>
+
+1. Sending a JavaScript alert from a Kotlin function.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/02_js_function.md]: Sends a JavaScript alert from a Kotlin function

File: examples/09_Kotlin_JS/02_js_function.md [hunk 4]
```diff
@@ -19,6 +32,6 @@ fun main(){
 
 </div>
 
-1. Create a JavaScript object literal. The `js(...)` function return type is `dynamic`.
-2. Add some properties by utilizing the `dynamic` type capabilities.
-3. Pass the JSON to JavaScript API.
+1. Creating a JavaScript object literal. The `js(...)` function return type is `dynamic`.
+2. Adding some properties by utilizing the `dynamic` type capabilities.
+3. Passing the JSON to JavaScript API.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/02_js_function.md]: Creates
Adds
Passes

File: examples/09_Kotlin_JS/02_js_function.md [hunk 5]
```diff
@@ -5,6 +5,19 @@ Should be used with extreme care.
 
 <div class="language-kotlin" theme="idea" data-min-compiler-version="1.3" data-target-platform="js">
 
+```kotlin
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/02_js_function.md]: from the text above ^^:

You can inline JavaScript code into your Kotlin code using the `js("…")` function.
 This should be used with extreme care.

File: examples/09_Kotlin_JS/03_external.md [hunk 6]
```diff
@@ -14,8 +14,8 @@ fun main() {
 
 </div>
 
-1. Declare an existing JavaScript function `alert` which takes a single `String` argument
-2. Use it as if it was regular Kotlin.
+1. Declaring an existing JavaScript function `alert` which takes a single `String` argument.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/03_external.md]: Declares

File: examples/09_Kotlin_JS/03_external.md [hunk 7]
```diff
@@ -14,8 +14,8 @@ fun main() {
...
+2. Using `alert` as if it was regular Kotlin.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/03_external.md]: Uses 'alert' as if it were regular Kotlin

File: examples/09_Kotlin_JS/03_external.md [hunk 8]
```diff
@@ -14,8 +14,8 @@ fun main() {
...
 
 Note that Kotlin checks during compilation that exactly a single argument of type `String` is passed.
 That prevents a number of bugs even when using pure JavaScript API - same as with regular Kotlin.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/03_external.md]: It prevents some bugs even when using pure JavaScript API - it's the same as with regular Kotlin.

actually, I don't really know what the "same as with regular Kotlin" means here??
Maybe there is a clearer way to say it.

File: examples/09_Kotlin_JS/06_HtmlBuilder.md [hunk 9]
```diff
@@ -1,10 +1,8 @@
...
 
+Below is an example of a type-safe Groovy-style builder. In this example we show how to describe an HTML page in Kotlin.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/06_HtmlBuilder.md]: Below is an example of a type-safe Groovy-style builder. In this example, we will describe an HTML page in Kotlin.

File: examples/09_Kotlin_JS/01_dynamic.md [hunk 10]
```diff
@@ -30,9 +30,9 @@ fun main(){
 
 </div>
 
-1. Any value can be assigned to variable to `dynamic` type
-2. Dynamic value can be assigned to anything.
-3. Dynamic value can be passed as argument to any function
-4. Any property or function with any arguments can be called on `dynamic` value 
-5. Dynamic call always returns dynamic value, so it is possible to chain them.
-6. Operators, assignments and indexed access (`[..]`) are translated "as is". Beware!
+1. Any value can be assigned to variable of the `dynamic` type.
```
Comment (B1ggDave) [inline examples/09_Kotlin_JS/01_dynamic.md]: Any value can be assigned to a `dynamic` variable type.

--is this right?

File: examples/09_Kotlin_JS/03_external.md (hunk 8 above)
Comment (p7nov) [reply examples/09_Kotlin_JS/03_external.md]: I'll remove with part. The previous sentence says enough about argument checking and it's common for Kotlin.

File: examples/09_Kotlin_JS/01_dynamic.md (hunk 10 above)
Comment (p7nov) [reply examples/09_Kotlin_JS/01_dynamic.md]: Yes, type checks aren't performed for dynamic.

**Pull request merged**